### PR TITLE
create Icons & Splash Screen

### PR DESCRIPTION
**Pull request by yehorDiakonov**

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 import {store} from './src/store/reducers';
 
-export const App = () => (
-  <Provider store={store}>
-    <RootStack />
-  </Provider>
-);
+export const App = () => {
+  React.useEffect(() => {
+    SplashScreen.hide();
```
Comment (BogdanKusch) [inline App.tsx:10]: we need hide splash after we will understand that: user is authorised or not.

File: ios/Podfile.lock [hunk 2]
```diff
@@ -205,6 +205,8 @@ PODS:
   - React-jsinspector (0.64.0)
```
Comment (BogdanKusch) [inline ios/Podfile.lock:205]: you can add this file to .gitignore

**Review by BogdanKusch — CHANGES_REQUESTED** [review]

File: ios/test.swift [hunk 3]
```diff
@@ -0,0 +1,8 @@
+//
```
[the remark this answers is not in the document]
Comment (yehorDiakonov) [reply ios/test.swift:1]: I was create this file manually because i have some errors, that file solved that

File: App.tsx [hunk 4]
```diff
@@ -1,11 +1,44 @@
...
-export const App = () => (
-  <Provider store={store}>
-    <RootStack />
-  </Provider>
-);
+export const App = () => {
+  const [isLoading, setIsLoading] = React.useState(true);
+
+  const userIsAuth = React.useCallback(async () => {
+    const status = (await AsyncStorage.getItem('status')) === 'true';
```
Comment (BogdanKusch) [inline App.tsx:15]: No need to do this, wait till youll receive result from async func then do what you need.

**Review by BogdanKusch — CHANGES_REQUESTED** [review]

**Review by BogdanKusch — APPROVED** [review]

**Pull request merged**